src/lib/Components/Artist/ArtistHeader.tsx

```diff
@@ -33,20 +30,39 @@ class Header extends React.Component<Props, State> {
   }
 
   render() {
-    const artist = this.props.artist
-    return (
-      <View style={{ paddingTop: 20 }}>
-        <Headline style={[styles.base, styles.headline]}>{artist.name}</Headline>
-        {this.renderByline()}
-        {this.renderFollowersCount()}
-        {this.renderFollowButton()}
-      </View>
-    )
-  }
+    const { artist } = this.props
+    const count = this.state.followersCount
+    const followerString = count === 1 ? " Follower" : " Followers"
+    const bylineRequired = artist.nationality || artist.birthday
 
-  renderFollowButton() {
-    if (this.props.artist.isFollowed !== null) {
-      return (
+    return (
+      <Box px={2} pt={3}>
+        <Serif style={{ textAlign: "center" }} size="5">
+          {artist.name}
+        </Serif>
+        <Spacer mb={0.5} />
+        {(count || bylineRequired) && (
+          <>
+            <TextWrapper style={{ textAlign: "center" }}>
+              {bylineRequired && <Sans size="2">{this.descriptiveString()}</Sans>}
+              {count &&
```

Comment:
If `count` is non-nil, but `bylineRequired` is, then this could cause a crash because the expression will be evaluated as `count`, I think? We should add `!!` in front of non-boolean variables when we want to explicitly evaluate them as booleans.

Reply:
Cool, just added some coercion here.